doc/release-notes/2.2.md

```diff
@@ -1,10 +1,65 @@
 # 2.2 Release Notes
 
+Version 2.2.0 of the Npgsql Entity Framework Core provider has been released and is available on nuget. This version works with [version 2.2.0 of Entity Framework Core](https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-2.2), and contains some new Npgsql features as well.
```

Comment:
I thought it used to be possible to omit "en-us" from the docs links so that the user's locale/region would be used to redirect them to their local language docs.

Though of course, _our_ docs are English-only....but something to consider.

Reply:
Good idea! Done here and everywhere else.